cvat-ui/src/components/annotation-page/standard-workspace/objects-side-bar/objects-side-bar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import {
+    Icon,
+    Layout,
+} from 'antd';
+
+interface State {
+    collapsed: boolean;
+}
+
+export default class StandardWorkspaceComponent extends React.PureComponent<{}, State> {
+    public constructor(props: any) {
+        super(props);
+        this.state = {
+            collapsed: true,
+        };
+    }
+
+    public render(): JSX.Element {
+        const { collapsed } = this.state;
+        return (
+            <Layout.Sider
+                className='cvat-annotation-page-objects-sidebar'
+                theme='light'
+                width={300}
+                collapsedWidth={0}
+                reverseArrow
+                collapsible
+                trigger={null}
+                collapsed={collapsed}
+            >
+                {/* eslint-disable-next-line */}
+                <span
+                    className={`cvat-annotation-page-objects-sidebar
+                        ant-layout-sider-zero-width-trigger
+                        ant-layout-sider-zero-width-trigger-left`}
+                    onClick={
+                        (): void => this.setState({ collapsed: !collapsed })
```

Comment:
It's better to use function when you rely on [previous state](https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous):
```javascript
this.setState(prevState => ({ collapsed: !prevState.collapsed }))
```

Reply:
Interesting, thx